### PR TITLE
docs: Fixed typos, comment style, and word wrapping

### PR DESCRIPTION
**Pull request by kyranet**

### CODE REVIEW

**Review by vladfrangu — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -15,41 +15,60 @@ resources:
...
-This template will install the specified version of node.js, install dependencies, and run the `test:lint` script defined in your package.json. (meaning you can use any lint tool or any combination of tools)
+This template will install the specified version of Node.js, install dependencies, and run the `test:lint` script
+defined in your `package.json` (meaning you can use any linter or any combination of tools).
 
 ```yml
 jobs:
   - template: lint.yml@templates
-    parameters: # Optional if you want to use all defaults
-      versionSpec: '10.x' # Optional (default: '10.x'): The version of node.js you want to use, valid values are ['8.x', '9.x', '10.x', '11.x', ...]
+    # Optional: Whether or not you want to use all defaults
```
Comment (bdistin) [inline README.md]: the original text is more accurate, it's not optional if you do not want to use all defaults

File: README.md [hunk 2]
```diff
@@ -15,41 +15,60 @@ resources:
       endpoint: dirigeants # This is likely your github username or organization name
 ```
 
-**Note:** You will have to allow your pipeline to use your github credentials to be used to fetch templates from other repos.
-You can do this by going to your Project Settings (in azure devops):
+> **Note:** You will have to allow your pipeline to use your github credentials to be used to fetch templates from other
+repos.
+
+You can do this by going to your Project Settings (in Azure DevOps):
```
Comment (bdistin) [inline README.md]: these lines are still a part of the note, idk if you can do the ordered list inside the block quote though

**Review by bdistin — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kyranet) [reply README.md]: Reverted in 07c583d

File: README.md (hunk 2 above)
Comment (kyranet) [reply README.md]: Fixed in 07c583d

Yeah, looks like we can block quote them as long as you put a `>` at the start of every line.

**Review by bdistin — APPROVED** [review]

**Pull request merged**